android/src/main/java/com/marigold/rnsdk/RNMarigoldPackage.kt

```diff
@@ -0,0 +1,38 @@
+package com.marigold.rnsdk
+
+import androidx.annotation.VisibleForTesting
+import com.facebook.react.ReactPackage
+import com.facebook.react.bridge.JavaScriptModule
+import com.facebook.react.bridge.NativeModule
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.uimanager.ViewManager
+import com.marigold.sdk.EngageBySailthru
+import com.marigold.sdk.Marigold
+
+class RNMarigoldPackage : ReactPackage {
+
+    var displayInAppNotifications = true
+
+    @VisibleForTesting
+    private val marigold = Marigold()
+
+    @VisibleForTesting
+    private val engageBySailthru = EngageBySailthru()
```

Comment:
I don't think this is used anywhere either, and would throw exceptions for non-ST customers - we should definitely remove this one.